examples/express-app/server.mjs

```diff
@@ -0,0 +1,64 @@
+import express from "express";
+import path from "path";
+
+import { load } from "@azure/app-configuration-provider";
+const connectionString = "<your-connection-string>";;
```

Comment:
in examples codes, I would suggest including more comments to explain why or include suggestions. 

like this line, I would prefer to have comments saying connection string is for local quick start and is not encouraged.

Reply:
and need to remove extra trailing `;`